modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -123,18 +124,12 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
                 { $"kfc|keyframecount=|{StandaloneCliConfigKeys.DefaultKeyFrameCount}=",
                     "The default number of delta messages to send until a key frame message is sent. If 0, no key frame messages are sent, if 1, every message will be a key frame. \nDefault: `0`.\n",
                     (int i) => this[StandaloneCliConfigKeys.DefaultKeyFrameCount] = TimeSpan.FromMilliseconds(i).ToString() },
-                { $"msi|metadatasendinterval=|{StandaloneCliConfigKeys.DefaultMetaDataSendInterval}=",
+                { $"msi|metadatasendinterval=|{StandaloneCliConfigKeys.DefaultMetaDataUpdateTime}=",
                     "Default value in milliseconds for the metadata send interval which determines in which interval metadata is sent.\nEven when disabled, metadata is still sent when the metadata version changes unless `--mm=*Samples` is set in which case this setting is ignored. Only valid for network message encodings. \nDefault: `0` which means periodic sending of metadata is disabled.\n",
-                    (int i) => this[StandaloneCliConfigKeys.DefaultMetaDataSendInterval] = TimeSpan.FromMilliseconds(i).ToString() },
+                    (int i) => this[StandaloneCliConfigKeys.DefaultMetaDataUpdateTime] = TimeSpan.FromMilliseconds(i).ToString() },
                 { $"dm|disablemetadata:|{StandaloneCliConfigKeys.DisableDataSetMetaData}:",
                     "Disables sending any metadata when metadata version changes. This setting can be used to also override the messaging profile's default support for metadata sending. \nDefault: `False` if the messaging profile selected supports sending metadata, `True` otherwise.\n",
                     (bool? b) => this[StandaloneCliConfigKeys.DisableDataSetMetaData] = b?.ToString() ?? "True" },
-
-                // TODO: Default metadata output name
-
-                { $"ri|enableroutinginfo:|{StandaloneCliConfigKeys.EnableRoutingInfo}:",
-                    "Add the routing info to telemetry messages. The name of the property is `$$RoutingInfo` and the value is the `DataSetWriterGroup` for that particular message.\nWhen the `DataSetWriterGroup` is not configured, the `$$RoutingInfo` property will not be added to the message even if this argument is set.\nDefault: `False` (disabled).\n",
-                    (bool? b) => this[StandaloneCliConfigKeys.EnableRoutingInfo] = b?.ToString() ?? "True" },
                     { $"lc|legacycompatibility=|{StandaloneCliConfigKeys.LegacyCompatibility}=",
```

Comment:
I'm not sure if we're missing something like Default `MetaDataQueueName` here?

Reply:
Below in transport settings:  { $"mqn|metadataqueuename:|{StandaloneCliConfigKeys.DefaultDataSetMetaDataQueueName}:",